.gitignore

```diff
@@ -39,7 +39,7 @@
 npm-debug.log
 testem.log
 /typings
-
+package-lock.json
```

Comment:
THANK YOU for this :D

Reply:
well, It turns out that I can't delete this file, because our bot search the file for vulnerabilities